### ISSUE TITLE
HCLS compliant data sets descriptions: summary description

### ISSUE THREAD

**Issue by egonw** [enanomapper/data.enanomapper.net#88]
... following the following standard: https://www.w3.org/TR/hcls-dataset/

MUST

- [ ] rdf:type dctypes:Dataset
- [ ] dct:title
- [ ] dct:description
- [ ] dct:publisher

where:

    dct:	http://purl.org/dc/terms/

**Comment by egonw** [comment]
Currently, the ENMRDF export has:

    <https://apps.ideaconsult.net/enmtest/bundle/1>
            a                     ot:Dataset ;
            dc:source             "http://dx.doi.org/10.6084/m9.figshare.1330208" ;
            dc:title              "NanoWiki" ;
            dcterms:rights        <https://creativecommons.org/publicdomain/zero/1.0/> ;
            dcterms:rightsHolder  <http://orcid.org/0000-0001-7542-0286> .

where:

    dc:    <http://purl.org/dc/elements/1.1/> .
    dcterms: <http://purl.org/dc/terms/>

**Comment by vedina** [comment]
Yes, these come from the OpenTox dataset API

**Comment by egonw** [comment]
So, changes needed to be made:

1. the Dataset type must be added or replacing the current type
2. the title is given using a different DC namespace
3. description is missing
4. publisher is missing

**Comment by egonw** [comment]
After talking with @vedina, I decided to update AMBIT to support both exports. For the RDF import, I will stick to the new HCLSig specification.

**Comment by vedina** [comment]
@egonw look for classes descendant of `QueryRDFReporter`

**Comment by vedina** [comment]
Can we close if this is implemented in the RDF export now ?

**Comment by egonw** [comment]
no, not yet full done. I'll leave this open (and plan to continue working on it) as this is related to work in other projects.
